modules/document_repository/README.md

```diff
@@ -0,0 +1,46 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a tool that provides a centralized location for non-candidate documents relevant to the study. For example, copies of non-filled instruments, publications, manual of operations.
+
+## Intended Users
+
+ LORIS users can view, download/upload files, edit information about files, and delete files given appropriate permissions.
```

Comment:
suggestion: "Given the appropriate permissions, LORIS users can view, download, upload, and delete files, as well as edit information pertaining to the files."